### ISSUE TITLE
Item user - Not outputting correctly

### ISSUE THREAD

**Issue by Hasan2143** [PrinceOfAmber/Cyclic#2216]
Minecraft Version: 
1.18.2

Forge Version: 
40.1.84

Mod Version: 
1.18.2 - 1.7.13

Single Player or Server:
Single Player

Describe problem (what you were doing; what happened; what should have happened):

Tested a variety of mod cables including Cyclic's own cables .I also tried every side of the block but, no matter what, the item inside the Item user block would not output.

Log file link:

Video/images/gifs (direct upload or link):

**Comment by HalcyonSeas73** [comment]
More info needed; what are you trying to use in the item user?

**Comment by Hasan2143** [comment]
Trying to use a bucket to right click on a cow to get milk. The right click
operation works fine but the milk bucket just won't output.

**Comment by HalcyonSeas73** [comment]
> Trying to use a bucket to right click on a cow to get milk. The right click operation works fine but the milk bucket just won't output.

Photo of your setup? Is the milk bucket just sitting in the Item User inventory?

**Comment by Hasan2143** [comment]
Yes the bucket just sits in the item users inventory, I can't send a
picture atm but if you really need one I can send it.

**Comment by Dan3a** [comment]
Still a problem for me on 1.19.2.
Trying to automate a Petal Apothecary but the empty bucket won't output.

**Comment by TBaldy** [comment]
Similar issue happening with the Combustion Generator. It accepts lava buckets and uses them as fuel, but then can't output the bucket from any side of the machine using hoppers, Pipez's Item Pipes and there don't appear to any Cyclic cables included with this version of the mod.

Minecraft Version: 1.19.2

Forge Version: 43.3.5

Mod Version: 1.19.2-1.8.2

**Comment by kollinkoko** [comment]
Still an issue on 1.20.1 (Combustion Generator)
